src/utils/newsletter.ts

```diff
@@ -14,15 +14,28 @@ export type NewsletterPayload = {
     action: SubscriptionStatus;
 };
 
-export interface NewsletterHandler {
-    (payload: NewsletterPayload): Promise<SubscriptionStatus>;
+export interface NewsletterUpdater {
+    (email: NewsletterPayload): Promise<SubscriptionStatus>;
```

Comment:
I would name it newsletterPayload instead because it's not just email in it

Reply:
Yes I already changed it back to payload (you were on an outdated commit)